### DISCUSSION TITLE
pypy with gunicorn

### DISCUSSION THREAD

**Discussion by lokeish** [falconry/falcon#1910] [Q&A]
How to start the gunicorn with pypy to run the falcon.

[a comment by an automated account is not shown]

**Comment by CaselIT** [comment]
Hi,

I don't think there's anything special to do, just install gunicorn under your pypy environment and run the application as you normally would. Depending on the application you may see significant performance gains by using the [gevent workers](https://docs.gunicorn.org/en/stable/settings.html#worker-class) of gunicorn.

**Comment by vytas7** [comment]
A better deployment guide for Gunicorn+PyPy is also tracked as https://github.com/falconry/falcon/issues/1369.
